### PR TITLE
Show model not found message

### PR DESCRIPTION
**Pull request by nanaya**
The model naming and the way it's done don't seem quite right but 🤷

Also I hope crowdin doesn't choke on the key name...

Related: ppy/osu#27720

### CODE REVIEW

File: app/Exceptions/Handler.php [hunk 1]
```diff
@@ -90,6 +90,29 @@ private static function isOAuthSessionException(Throwable $e): bool
...
+            'model' => trans_exists($modelTransKey, $GLOBALS['cfg']['app']['fallback_locale'])
+                ? osu_trans($modelTransKey)
+                : trim(strtr($model, ['App\Models\\' => '']), '\\'),
+        ];
+
+        if (empty($ids)) {
+            $key = 'models.not_found_no_ids';
+        } else {
+            $key = 'models.not_found';
+            $params['ids'] = implode(', ', $ids);
```
Comment (nanaya) [inline app/Exceptions/Handler.php:110]: I wonder if the id shouldn't be visible. I took it from the original laravel exception message

Comment (notbakaneko) [reply app/Exceptions/Handler.php:110]: I'm thinking it shouldn't be shown in general, but in the context of the other issue, it'd make sense to know which one is missing? 🤔

**Review by notbakaneko — CHANGES_REQUESTED** [review]
Yeah, I'm not sure if this should become the default behaviour...

image

File: app/Exceptions/Handler.php [hunk 2]
```diff
@@ -90,6 +90,22 @@ private static function isOAuthSessionException(Throwable $e): bool
             && $e->getMessage() === 'Authorization request was not present in the session.';
     }
 
+    private static function modelNotFoundMessage(ModelNotFoundException $e): string
+    {
+        $ids = $e->getIds();
```
Comment (notbakaneko) [inline app/Exceptions/Handler.php:95]: now it's unused 👀

**Review by notbakaneko — CHANGES_REQUESTED** [review]

**Review by notbakaneko — APPROVED** [review]

**Pull request merged**